robot-server/robot_server/instruments/instrument_models.py

```diff
@@ -81,13 +81,23 @@ class PipetteData(BaseModel):
     #  add calibration data as decided by https://opentrons.atlassian.net/browse/RSS-167
 
 
+class PipetteState(BaseModel):
+    """State from an attached pipette."""
+
+    tip_detected: bool = Field(
```

Comment:
🐫-case for HTTP-facing field names.

```suggestion
    tipDetected: bool = Field(
```